### ISSUE TITLE
The club name in the Constitution PDF needs to be updated to Computer Science Group

### ISSUE THREAD

**Issue by zkhussain** [ius-csg/laws#8]
The club name in the Constitution PDF needs to be updated from Computer Security Group to Computer Science Group.

**Comment by Cliftonz** [comment]
It is the markdown file in this repository, that then needs propagated to the website.

**Comment by zkhussain** [comment]
I checked it, the file is in csghomepage/assets/pdf/Constitution.pdf. Do I delete and add the new file with updates or is there a way to edit it in here?

**Comment by Cliftonz** [comment]
Yes, a better way would be to link to the the latest release of the constitution or bylaws but I don't know where Adam is at on that build set for the repository.

**Comment by TheOtherUnknown** [comment]
Ta-da! https://github.com/ius-csg/laws/releases/tag/v1.0
